File: 01-variables/js/fase4.js
let nomComplet = "Jordi Calderón Santamaría";
let data = "05/07/1973";

let any = data.substr(6);

console.log("El meu nom és " + nomComplet);
console.log("Vaig nèixer el " + data);
console.log("El meu any de naixement " + (es_bisiesto(any) ? "" : "NO ") + "és de traspàs");



function es_bisiesto(year) {
    return year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
}

File: 01-variables/java/itinerari/variables/java/itinerari/variables/Fase4.java
package itinerari.variables;

public class Fase4 {
    public static void main(String[] args) {
        
        String nomComplet = "Jordi Calderón Santamaría";
        String data = "05/07/1973";

        int any = Integer.valueOf(data.substring(6));
        boolean esTraspas = Fase2.isLeap(any);

        System.out.println("El meu nom és " + nomComplet);
        System.out.println("Vaig nèixer el " + data);
        System.out.println("El meu any de naixement" + (!esTraspas ? " no " : "") + "és de traspàs." );

    }
}